Code/Framework/AzCore/AzCore/Script/ScriptSystemBus.h

```diff
@@ -14,6 +14,9 @@
 
 struct lua_State;
 
+// #define SCRIPT_SYSTEM_SCRIPT_STATUS(window, msg, ...) AZ_TracePrintf(window, msg, __VA_ARGS__);
+#define SCRIPT_SYSTEM_SCRIPT_STATUS(window, msg, ...)
```

Comment:
Used to diagnose the load status of ScriptAssets vs whether they are loaded in lua_State, which is needed when actively developing / debugging a new paradigm that uses them.

Reply:
done